code/modules/research/techweb/all_nodes.dm

```diff
@@ -222,7 +222,7 @@
 	id = "posibrain"
 	display_name = "Positronic Brain"
 	description = "Applied usage of neural technology allowing for autonomous AI units based on special metallic cubes with conductive and processing circuits."
-	prereq_ids = list("mmi", "neural_programming")

```

Comment:
i think this is justified

Reply:
Yeah, what is the rationale here?

Reply:
look at this in side-by-side mode @XDTM @KorPhaeron

Reply:
When i made the comment he had removed mmi, now it has been readded in reverse, so the diff does not appear outdated

Reply:
hacked